tests/unit/actions/clickable-test.js

```diff
@@ -1,62 +1,70 @@
 import { test } from 'qunit';
 import { moduleFor, buildProperty } from '../test-helper';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['buildProperty' is defined but never used.](https://www.codacy.com/app/san650/ember-cli-page-object/file/1817059231/issues/source?bid=2903707&fileBranchId=3076256#l2)